fuse_examples/classification/MG_CMMD/data_source_folds.py

```diff
@@ -0,0 +1,112 @@
+"""
```

Comment:
this one is generic, right?
lets move the file to fuse/data/data_source/

Reply:
done , added changes to the file on summary and toolbox